results/anonymized_text.txt

```diff
@@ -0,0 +1,3 @@
+
```

Comment:
Did you mean to commit this folder of results?

Unless it's required for the service I'd remove it from the repo to avoid clutter